setup.py

```diff
@@ -38,6 +38,16 @@
     + jarvis_deps
 )
 
+
+def myversion():
```

Comment:
How does this come into play?

Reply:
Good catch, I forgot to take out the setuptools_scm hacks.